aopy/postproc/base.py

```diff
@@ -185,22 +186,24 @@ def get_conditioned_trials_per_target(target_idx, min_trial, cond_mask=None, rep
         (ntr): trial index to extract the same number of conditioned trials for each target
         
     '''
-    if seed:
+    if seed is not None:
         np.random.seed(seed)
         
     # Get trial index to get the same number of trials per target
-    tmp = []
-    for itarget in np.unique(target_idx):
+    for idx, itarget in enumerate(np.unique(target_idx)):
         if cond_mask is None:
             trial_mask_targ = np.where(target_idx == itarget)[0]
         else:
             trial_mask_targ = np.where(cond_mask * (target_idx == itarget))[0] 
                    
         if trial_mask_targ.size:
-            tmp = np.concatenate([tmp, np.random.choice(trial_mask_targ, min_trial, replace=replacement)])
+            if idx == 0:
```

Comment:
Now the input arg "min_trial" is not an accurate description, I think you should rename it to something like "trials_per_cond"